_docs/install/kubernetes/dynamic-provisioning.md

```diff
@@ -31,14 +32,14 @@ changed its password (recommended), you must specify the new settings in
 `apiAddress`, `apiUsername` and `apiPassword`, encoded as base64 strings.
 
 ```bash
-$ echo -n "tcp://127.0.0.1:5705" | base64
+echo -n "tcp://127.0.0.1:5705" | base64
```

Comment:
maybe include a note here to explain that the k8s master needs to be able to access the api, so if the master doesn't run the storageos container then a LoadBalancer ip may be needed.  (I can give a sample yaml if needed)

Reply:
Added note. Yes please to the sample yaml.